integrationTests/cypress-setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+export NODE_ENV=development
+
+docker-compose exec api npm run migrate
```

Comment:
could we do this more like the endpoint-test/endpoint.sh and do a 
```
docker-compose up -d
```
before we run these other commands, just in case the api isn't up

Reply:
sure, I can do that.  It seems to do nothing if the containers are up already.

Reply:
Doing this causes the containers to come down at the end of the tests which is a different kind of annoying.